policy/executor/internal/nodes.go

```diff
@@ -315,6 +318,16 @@ func (nodeData *ReportingQueryNodeData) score() *policy.Score {
 			break
 		}
 
+		if cur.Data.Error != nil && strings.HasSuffix(cur.Data.Error.Error(), "not found") {
```

Comment:
i feel like it would be possible for a lot of different errors to have a "not found" error that doesn't mean the asset went away

Reply:
perhaps we can define some common errors and check for those. If the asset is disconnected, the provider should return that error type

Reply:
I've changed the error message where it is created to be more precise: https://github.com/mondoohq/cnquery/pull/361

I think, now that I have the point where the message is created, I could also create a specific error type out of it.

Reply:
Changed.